gsi/gss_assist/source/programs/grid-mapfile-add-entry.8

```diff
@@ -87,6 +87,11 @@ Modify the gridmap file named by
 instead of the default\&.
 .RE
 .PP
+*\-force
```

Comment:
I think this should read `\fB\-force\fR` when you compare it to the markup of `-dm -dryrun` further above. But maybe this is an artifact of the used "DocBook XSL Stylesheets v1.79.1" instead of what was used for the original file.

**UPDATE:** This is due to a missing `*` in https://github.com/gridcf/gct/pull/134/files#diff-970a287b14b8f63d7c4d1b1660881053. (see comment there!).

Reply:
I added your changes and now the tests finish successfully.